test/python/quantum_info/test_synthesis.py

```diff
@@ -1542,6 +1542,20 @@ def test_opt_a1a2(self, nqubits):
             ccirc.count_ops().get("cx"), (23 / 48) * 4**nqubits - (3 / 2) * 2**nqubits + 4 / 3
         )
 
+    # 10036
```

Comment:
Nit: I think a little more context here would be useful. Just seeing  `10036` it wasn't obvious at first that this was the issue number that the test was coming from. I think just making this a link would be more useful, especially coming back to this in a year or two and we're not so fresh on the issue number being fixed here.

```suggestion
    # From: https://github.com/Qiskit/qiskit-terra/issues/10036
```

Reply:
I was copying a common style for marking the bug report whose fix is being tested. I agree that with online code browsers and contemporary IDE's having it linkable would be great. In fact, GH is close to doing this automatically. It already makes the link in comments in PRs and issues. I would at least vote for a single style in qiskit (or at least terra) when people choose to mention the bug. For example always include the full url... But going further, I often ask myself "why is this test here" and see others discuss it too. And wonder if the test is made to ensure an "invariant" or even just enforce some API or design decision. It was the latter discussions that prompted me to add an issue/PR number.